<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonSettings</class>
 <widget class="QDialog" name="ButtonSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="enter">
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="mode"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="settings" native="true">
     <widget class="QWidget" name="slider" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QSpinBox" name="slider_to">
         <property name="minimum">
          <number>-10000</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="slider_label3">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="slider_input"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="slider_label2">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="slider_label">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="slider_from">
         <property name="minimum">
          <number>-10000</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="vertical">
         <property name="text">
          <string>Vetical</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="horizontal">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="button" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>394</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="button_input2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="button_label2">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="button_input"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="button_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Action: </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>enter</zorder>
  <zorder>mode</zorder>
  <zorder>settings</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
